res/controllers/common-controller-scripts.js

```diff
@@ -245,6 +245,89 @@ script.midiDebug = function(channel, control, value, status, group) {
         " status: 0x" + status.toString(16) + " group: " + group);
 };
 
+
+
+class StringDeburr {
+    constructor() {
+        this.regexLatin = /[\xc0-\xd6\xd8-\xf6\xf8-\xff\u0100-\u017f]/g;
```

Comment:
can you explain this regex a little more? From what I can tell, this doesn't match the layout on wikipedia. https://en.wikipedia.org/wiki/ISO/IEC_8859-1#Code_page_layout

Reply:
Eeeh… No I can't. Most of this code is copied from Lodash's `_.deburr`, which I trust :sweat_smile: